flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -126,12 +126,16 @@ public static void assertTableRows(String tablePath, List<RowData> expected) thr
     assertTableRecords(tablePath, expectedRecords);
   }
 
-  public static void assertTableRecords(String tablePath, List<Record> expected) throws IOException {
-    Preconditions.checkArgument(expected != null, "expected records shouldn't be null");
-    Table newTable = new HadoopTables().load(tablePath);
-    try (CloseableIterable<Record> iterable = IcebergGenerics.read(newTable).build()) {
+  public static void assertTableRecords(Table table, List<Record> expected) throws IOException {
+    table.refresh();
```

Comment:
Can this be done automatically when a write completes, or is this a completely separate copy of the table?

Reply:
Since we don't support to scan table by flink sql,   so we have to read records from iceberg table by iceberg Java API in unit tests.  In this [test](https://github.com/apache/iceberg/pull/1348/files#diff-6fc1fa50af5a93944270e7dd4c0ea393R98),  we get the `icebergTable` instance firstly, then the following test methods will commit the iceberg table by flink sql,  the `icebergTable` need a fresh to catch the latest changes.